js/plugins/evaluators/src/index.ts

```diff
@@ -75,15 +72,16 @@ export function genkitEval<
 
 export default genkitEval;
 
-function hasMetric(arr: GenkitMetric[] | undefined, metric: GenkitMetric) {
-  return arr?.some((m) => m === metric);
-}
-
-function fillScores(dataPoint: BaseEvalDataPoint, score: Score): EvalResponse {
-  return {
-    testCaseId: dataPoint.testCaseId,
-    evaluation: score,
-  };
+function fillScores(
+  dataPoint: BaseEvalDataPoint,
+  score: Score,
+  statusOverrideFn?: (s: Score) => EvalStatusEnum
```

Comment:
does this fn need to know the metric? and maybe original status as well? so that one could do something like:

```ts
statusOverrideFn = ({score, originalStatus, metric}) => {
  if (metric === 'faithfulness') {
    return ...;
  }
  return originalStatus;
}
```

Reply:
- originalStatus is available in the score object
- Since this is configured per metric, I think the metric field is redundant.

I like the idea of using an object for future expansion, so I added that.